lib/rules/lint-duplicate-attributes.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Rule = require('./base');
+
+const logDuplicateAttributes = function ({ attributes = [], indentifier, type }) {
+  let currentAttribute;
+  let currentIndex = 0;
+  let length = attributes.length;
+
+  attributes.forEach((attribute, index) => {
+    for (currentIndex = index; currentIndex < length; currentIndex++) {
+      currentAttribute = attributes[currentIndex];
+      if (attribute[indentifier] === currentAttribute[indentifier] && index !== currentIndex) {
+        this.log({
+          message: `Duplicate attributes found in the ${type}.`,
+          line: currentAttribute.loc && currentAttribute.loc.start.line,
+          column: currentAttribute.loc && currentAttribute.loc.start.column,
+          source: this.sourceForNode(currentAttribute)
```

Comment:
I think the source should be the node containing the attribute (not the attribute itself), what do you think?